jbmc/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ add_subdirectory(unit)
 
 add_custom_target(java-models-library ALL
     COMMAND mvn --quiet -Dmaven.test.skip=true package
```

Comment:
Not an action for this PR, but we should also avoid hardcoding the maven executable. By adding a configuration for this we could support cases where people want to build with `~/.local/bin/maven-experimental` or whatever.